helm/openidm/templates/openidm-secrets-env.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2016-2017 ForgeRock AS. Use of this source code is subject to the
+# Common Development and Distribution License (CDDL) that can be found in the LICENSE file
+# Secrets for OpenIDM stack deployment.
+# Note that secret values are base64-encoded.
+apiVersion: v1
+kind: Secret
+metadata:
+   name: "openidm-secrets-env"
+type: Opaque
+data:
+   OPENIDM_ADMIN_PASSWORD: {{ default "openidm-admin" .Values.secret.openidmAdminPassword | b64enc }}
```

Comment:
we should talk to Neil and see how we get rid of the need for passing in this secret value. This feels wrong.

Reply:
Previously this was in a configmap. I figured it was better to have as a secret.